File: Looping/Loop inside loop/Other Lang/C/LoopInLoop4.c
#include <stdio.h>

void main() {
	for (int vert = 1; vert <= 3; vert++) {
		//Nested Loops Gave Me Anxiety...
		for (int horz = 1; horz <= 3; horz++) {
			printf("%d x %d = %d\n", vert, horz, (vert * horz));
		}
		printf("\n");
	}
}

File: Looping/Loop inside loop/Other Lang/Python/LoopInLoop4.py
def main():
	for vert in range(1,4):
		# Nested Loops gave me anxiety, especially here...
		for horz in range(1,4):
			print(''.join([str(vert), " x ", str(horz), " = ", str(vert * horz)]))
		print("");

main()
